vuu-ui/showcase/src/examples/Apps/NewTheme.examples.tsx

```diff
@@ -56,29 +76,24 @@ export const ShellWithNewTheme = () => {
         }
         return menuDescriptors;
       },
-      (action: MenuActionClosePopup) => {
+      async (action: MenuActionClosePopup) => {
         console.log("menu action", {
           action,
         });
         if (action.menuId === "save-layout") {
```

Comment:
I've pushed a small change to main. In this MenuActionClosePopup 'action', you have an options object, populated from the Tab at the time the ContextMernu is created. Previoulsy, it just contained the index of the tab from which the popupMenu was launched. Now it also contains the controlledComponentId - the id of the associated TabPanel (its also visible in the html as the aria attribute 'aria-controls'.

Reply:
Brill thanks Steve. 

[aa898c2](https://github.com/ScottLogic/finos-vuu/pull/23/commits/aa898c20d1ff7f17315d89bf0996fa47f13c5518)